lib/sources/link_factory.cpp

```diff
@@ -6,33 +6,34 @@
 
 using namespace loodsman;
 
-int loodsman::factory(LinkPtr& linkOut ,link_type type, int local_port, std::string local_address, int remote_port,
-                           std::string remote_address)
+int loodsman::factory(LinkPtr& linkOut, link_type type, int local_port, std::string local_address,
```

Comment:
return link object

Reply:
Turn it to class. LinksFactory or LinksCreator

Reply:
I'll delete old factory after finishing this one